src/components/Articles.jsx

```diff
@@ -0,0 +1,33 @@
+import { fetchTopics } from "../utils/api";
+import { loadingMsg } from "../utils/messages";
+
+import ListArticles from "./ListArticles";
+
+
+import { useState, useEffect } from "react"
+
+
+
+function Articles () {
+
+  const [ isLoading, setIsLoading ] = useState(true)
+  const [ topics, setTopics ] = useState({})
+  const [ orderState, setOrderState ] = useState("asc")
+
+
+    return (
+      <>
+
+
+      <h3>Articles</h3>
+
+
+
+      <ListArticles />
```

Comment:
I think this second layer of nesting components is unnecessary, and is also going to confuse things. If you think about the useEffect happening in your ListArticles component, it's set to trigger whenever the page is refreshed, but that only refers to the list component, not the whole articles endpoint.

Reply:
There will be other components in "Articles.js", for examples, category menu, sort by options etc.
Therefore several states (e.g. category selected). I would keep this.